examples/4337-gas-metering/utils/safe.ts

```diff
@@ -165,45 +167,46 @@ const getInitializerCode = async ({
     })
   }
 
-  const multiSendCallData = encodeMultiSend(setupTxs)
+  const recipient = setupTxs.length > 1 ? multiSendAddress : safeModuleSetupAddress
```

Comment:
Previously, it'd execute the module setup through a multisend, even if there was one transaction. I added a condition that it only does so if we need more than one setup transaction.